themes/api.jquerymobile.com/index.php

```diff
@@ -41,10 +42,13 @@
 			category from the sidebar.</p>
 
 		<p>jQuery Mobile <?php echo $thisVersion; ?>
-			supports jQuery <?php echo $versions[ $thisVersion ]; ?>.</p>
+		<?php if ( $thisVersion == "1.5" ) {
```

Comment:
@scottgonzalez i have mixed feelings about how to best present this we automatically include certain parts of UI and explicitly are not compatible with others ( like the css ) in 1.5 we have aside from css about 95% compatibility the only remaining issue is we both have widgets named selectmenu. How ever there was far less support in previous versions and many more caveats to that support. I almost feel like we need a page specifically explaining how to work with the 2 together. we absolutely can and should document which version is imported in each though so maybe this is a separate issue.